### PR TITLE
CSU-NOTICKET: Update branding menu

### PR DESCRIPTION
**Pull request by mcortes19**
## Purpose:
- Description of PR

### Ticket(s)
[CSU-XX](https://fourkitchens.clickup.com/t/36718269/CSU-XX)

### Required by

- https://github.com/fourkitchens/s4-app/pull/22
- https://github.com/fourkitchens/s4-app-faculty/pull/1
- https://github.com/fourkitchens/s4-app-partners/pull/15

### Pull Request Deployment:
- Normal deployment process (`npm run rebuild` on local and ci on github)

### Functional Testing:
- [x] Go to the [Branding Bar](http://localhost:6006/?path=/story/components-branding-bar--default) and confirm the help button opens a modal

### CODE REVIEW

File: src/components/02-components/branding-bar/branding-bar.stories.tsx [hunk 1]
```diff
@@ -11,19 +11,16 @@ export default meta;
...
       { title: 'Student', url: '#' },
       { title: 'Faculty', url: '#', isActive: true },
       { title: 'Partner', url: '#' },
     ],
-    helpLink: '#',
     myAccountLink: '#',
     logoutLink: '#',
+    dialogTitle: 'Students',
+    dialogText:
+      '<p>For assistance please contact the campus program office(s) with which you are working. If you are having technical issues, please contact the <a href="mailto:s4help@calstate.edu ">Help Desk</a></p>',
```
Comment (ccjjmartin) [inline src/components/02-components/branding-bar/branding-bar.stories.tsx:24]: @mcortes19 I don't know if it matters but there is an extra space at the end of the email address for the mailto link and when I click the mailto link it doesn't actually do anything ...

**Review by ccjjmartin — APPROVED** [review]
@mcortes19 One thing to double check otherwise :shipit:

File: src/components/02-components/branding-bar/branding-bar.stories.tsx (hunk 1 above)
Comment (mcortes19) [reply src/components/02-components/branding-bar/branding-bar.stories.tsx]: @ccjjmartin great catch, I did  push the fix

Comment (ccjjmartin) [reply src/components/02-components/branding-bar/branding-bar.stories.tsx]: @mcortes19 🚢 The mailto link does work, I just found it on a different screen

**Pull request merged**